examples/ecs/command_queue_in_world.rs

```diff
@@ -0,0 +1,43 @@
+//! This example shows how to create a command queue and store it in the world.
+//! This can be useful for conditionally applying a command queue.
+
+use bevy::{
+    ecs::{entity::Entities, system::CommandQueue},
+    prelude::*,
+};
+
+// new typed command queue
+#[derive(Resource, Default)]
+struct MyCommandQueue(pub CommandQueue);
+
+// marker component for spawned entity
+#[derive(Component)]
+struct EntityMarker;
+
+fn main() {
+    App::new()
+        .init_resource::<MyCommandQueue>()
+        .add_system(spawn_something)
+        .add_system(apply_queue.at_end())
+        .add_system_to_stage(CoreStage::PostUpdate, log_entity)
+        .run();
+}
+
+fn spawn_something(entities: &Entities, mut q: ResMut<MyCommandQueue>) {
```

Comment:
`Entities` is a fairly niche system param, so I think this example could use a quick explanation of what it is.